src/utils/git.ts

```diff
@@ -47,3 +47,34 @@ export const getStagedGitDiff = async (
     diff
   };
 };
+
+export const getStagedFiles = async (): Promise<string[]> => {
```

Comment:
this is same as `getStagedGitDiff` line 21, but i would refactor it, so it's really a `get` function. now it's also staging files. could you refactor it to something like `getAndSetStagedDiff`? using your `gitAdd` function

Reply:
I agree with your assessment that it might not be necessary to create this new function. Instead, you propose to replace the call to getStagedGitDiff with await gitAdd({ files: await getChangedFiles() }).

This approach would stage all changed files and avoid the need for a separate function to retrieve and stage the diff. This seems like a reasonable and simpler solution, so I think it's a good idea to go ahead with this refactor.